doc/source/analyzing/fields.rst

```diff
@@ -604,26 +608,26 @@ as detailed below (see :ref:`field_parameters` for more information).
 
 The relative fields which are currently supported for gas fields are:
 
-* ``("gas", "relative_velocity_{xyz}")``, defined by setting the
-  ``"bulk_velocity"`` field parameter
-* ``("gas", "relative_magnetic_field_{xyz}")``, defined by setting the
-  ``"bulk_magnetic_field"`` field parameter
+* ``('gas', 'relative_velocity_x'')``, defined by setting the
+  ``'bulk_velocity'`` field parameter
+* ``('gas', 'relative_magnetic_field_x'')``, defined by setting the
```

Comment:
Looks like there is an extra `'` here

Reply:
good catch, I think my text editor ships those in pairs...

Reply:
done